crates/ironrdp-server/src/encoder/bitmap.rs

```diff
@@ -20,7 +20,13 @@ impl BitmapEncoder {
     }
 
     pub(crate) fn encode(&mut self, bitmap: &BitmapUpdate, output: &mut [u8]) -> Result<usize, PduError> {
-        let row_len = usize::from(bitmap.width.get()) * usize::from(bitmap.format.bytes_per_pixel());
+        if bitmap.width.get() % 4 != 0 {
+            // FIXME
```

Comment:
suggestion: It would be helpful to document such things with an inline code comment in addition to the commit message which is almost only read when reviewing the patch, but not, i.e., one year later.

```rust
// FIXME: support non-multiple of 4 widths.
// 
// It’s not clear how to achieve that yet, but generally, server uses multiple of 4 widths,
// and client has surface capabilities, so this path is unlikely.
```

Reply:
ack, thanks